app/controllers/about_us_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AboutUsController < ApplicationController
```

Comment:
I would say that adding an about page in this feature is out of scope

Reply:
Okay, but without navigation, why is there a feature about this? Since it's about navigation. I don't really get what this feature is without a second page?

Reply:
You don't need a controller to navigate to different pages, you can reuse the products controller. 
https://stackoverflow.com/questions/18703349/how-to-add-a-new-action-to-the-existing-controller

That being said why do you start a story about navigating if there is nothing to navigate to?

Reply:
ahhh nice. I thought you did need one. That clears things up.

There is something to navigate to, to about us.